app/src/org/commcare/network/CommcareRequestGenerator.java

```diff
@@ -225,6 +225,7 @@ public Response<ResponseBody> postMultipart(String url, List<MultipartBody.Part>
                 CommCareApplication.instance(),
                 url,
                 queryParams,
+                ArrayListMultimap.create(),

```

Comment:
what about making these `ImmutableMultiMap.of()` instead of creating multiple maps?